### PR TITLE
Restore `highlight-collaborators-and-own-discussions`

### LINKED ISSUE #2690: `highlight collaborators and own discussions` stopped working
No errors

image

### PR DESCRIPTION
**Pull request by yakov116**
fixes #2690

### CODE REVIEW

**Comment by yakov116** [comment]
@fregante I don't know type script 😄

File: source/features/highlight-collaborators-and-own-discussions.tsx [hunk 1]
```diff
@@ -11,8 +11,8 @@ async function highlightCollaborators(): Promise<false | void> {
 	}
 
 	const dom = await fetchDom(getRepoURL() + '/issues/show_menu_content?partial=issues/filters/authors_content');
-	const collaborators = select.all('.select-menu-item-text', dom).map(collaborator => {
-		return collaborator.firstChild!.textContent!.trim();
+	const collaborators = select.all('.SelectMenu-item [alt]', dom).map(collaborator => {
```
Comment (jerone) [inline source/features/highlight-collaborators-and-own-discussions.tsx]: You should cast it to `HTMLImageElement`:

```suggestion
	const collaborators = select.all<HTMLImageElement>('.SelectMenu-item [alt]', dom).map(collaborator => {
```

Comment (yakov116) [reply source/features/highlight-collaborators-and-own-discussions.tsx]: @jerone thanks

**Comment by yakov116** [comment]
@jerone what does the next error mean?

**Comment by jerone** [comment]
> @jerone what does the next error mean?

It means you have to remove the bang `!` on line 15, char 10.

**Comment by jerone** [comment]
@yakov116 You know you can run this yourself, without depending on GitHub Actions, right?
Open console on the extension root and run `npm run lint`.

**Comment by yakov116** [comment]
@jerone I know, I am not at a local computer.

**Comment by fregante** [comment]
Looking good! Thanks Yakov and Jeroen ^_^

**Pull request merged**